drupal/recipes/wunder_next_setup/config/next.next_site.frontend.yml

```diff
@@ -6,3 +6,5 @@ label: 'Next.js frontend'
 base_url: 'http://localhost:3000'
 preview_url: 'http://localhost:3000/api/preview'
 preview_secret: null
+revalidate_url: 'http://localhost:3000/api/revalidate'
```

Comment:
This configuration is for now empty, I have made a new story to actually activate this functionality: https://wunder.atlassian.net/browse/NEX-26